src/aria/widgets/form/DropDownTrait.js

```diff
@@ -167,11 +167,53 @@ module.exports = Aria.classDefinition({
             return this._frame && this._frame.getIcon ? [this._frame.getIcon("dropdown")] : null;
         },
 
+        /**
+         * Removes role="dialog" from all parent elements and stores those elements in this._waiDialogRoleRemoved.
+         */
+        _removeDialogRole : function () {
+            if (!this._cfg.waiAria) {
+                return;
+            }
+            var waiDialogRoleRemoved = this._waiDialogRoleRemoved;
+            if (!waiDialogRoleRemoved) {
+                this._waiDialogRoleRemoved = waiDialogRoleRemoved = [];
+            }
+            var domElt = this.getDom().parentElement;
+            while (domElt) {
+                if (domElt.getAttribute("role") === "dialog") {
+                    domElt.removeAttribute("role");
+                    waiDialogRoleRemoved.push(domElt);
+                }
+                domElt = domElt.parentElement;
+            }
+        },
+
+        /**
+         * Asynchronously restores role="dialog" on all elements in this._waiDialogRoleRemoved
+         * (and removes those elements from this array).
+         */
+        _restoreDialogRole : function () {
+            var waiDialogRoleRemoved = this._waiDialogRoleRemoved;
+            if (!waiDialogRoleRemoved || waiDialogRoleRemoved.length === 0) {
+                return;
+            }
+            setTimeout(function () {
```

Comment:
Why do we need the 1s delay here ?

Reply:
I reused the same delay as in the work done by @ymeine.
I tried to use setTimeout with no delay, but in that case, it randomly fails, probably because the focus of the field, which is asynchronous in IE, is sometimes done after restoring `role="dialog"` on the parent element.